jwst/lib/set_telescope_pointing.py

```diff
@@ -928,6 +928,8 @@ def calc_transforms(t_pars: TransformParameters):
     transforms : `Transforms`
         The list of coordinate matrix transformations
     """
+    t_pars.method = t_pars.method if t_pars.method else Methods.default
```

Comment:
It'd be better for TransformationParameters.method to be initialized with Methods.default rather than have users of the class figure out a default unless there's a special value needed for a particular situation.

Reply:
Good catch. This was leftover from having to deal with two different methods for COARSE vs. TRACK. But that has been handled, so a proper default can be added.

Reply:
You should now be able to take out this line.